content/solutions/altinn-studio/functional/ui-designer/components/index.md

```diff
@@ -132,10 +132,23 @@ weight: 100
           <li>Label text</li>
           <li>Description text</li>
           <li>Read only</li>
+        </ul>
       </td>
       <td><img alt="Text area component - edit properties" src="props-text-area.PNG?width=700" title="Text area component - edit properties" /></td>
       <td><img alt="Text area component - default look" src="display-text-area.PNG?width=700" title="Text area component - default look" /></td>
     </tr>
+    <tr>
+      <td>Datepicker</td>
+      <td>
+        <ul>
+          <li>Link to data model</li>
+          <li>Label text</li>
+          <li>Description text</li>
+        </ul>      
+      </td>
+      <td><img alt="Text area component - edit properties" src="props-datepicker.PNG?width=700" title="Datepicker component - edit properties" /></td>
+      <td><img alt="Text area component - default look" src="display-datepicker.PNG?width=700" title="Datepicker component - default look" /></td>
```

Comment:
Does it work with .PNG ending, when the file is saved as .png?

Reply:
~~what .PNG ending...?~~
**Fixed**